projects/plugins/jetpack/extensions/blocks/subscriber-login/editor.js

```diff
@@ -26,3 +27,14 @@ registerJetpackBlockFromMetadata( metadata, {
 		],
 	},
 } );
+
+addFilter( 'blocks.registerBlockType', 'jetpack-subscriber-login-nav-item', ( settings, name ) => {
+	if ( name === 'core/navigation' ) {
+		return {
+			...settings,
+			allowedBlocks: [ ...( settings.allowedBlocks ?? [] ), 'jetpack/subscriber-login' ],
```

Comment:
Did older versions of core block have `allowedBlocks`? If not, would be safer to do `settings?.allowedBlocks ?? []`

Reply:
If there is no `allowedBlocks` field in `settings`, `settings.allowedBlocks` will be `undefined`, so `settings.allowedBlocks ?? []` will evaluate to `[]`.